include/mata/nfa-algorithms.hh

```diff
@@ -21,10 +21,6 @@
 #include <mata/nfa.hh>
 #include <mata/simlib/util/binary_relation.hh>
 
-namespace Mata {
-
-namespace Nfa {
-
 /**
```

Comment:
Please check, that by running `make docs`, in generated doxygen files this do not screw things out.

Reply:
It works as expected.